docs/api/frame.rst

```diff
@@ -242,16 +293,30 @@
     .cbind()         <frame/cbind>
     .colindex()      <frame/colindex>
     .copy()          <frame/copy>
+    .countna()       <frame/countna>
+    .countna()       <frame/countna1>
     .export_names()  <frame/export_names>
     .head()          <frame/head>
     .key             <frame/key>
     .keys()          <frame/keys>
     .ltypes          <frame/ltypes>
     .materialize()   <frame/materialize>
+    .max()           <frame/max>
+    .max1()          <frame/max1>
+    .mean            <frame/mean>
+    .mean1           <frame/mean1>
     .meta            <frame/meta>
+    .min             <frame/min>
+    .min1            <frame/min1>
+    .mode            <frame/mode>
+    .mode1           <frame/mode1>
     .names           <frame/names>
     .ncols           <frame/ncols>
+    .nmodal          <frame/nmodal>
+    .nmodal1         <frame/nmodal1>
     .nrows           <frame/nrows>
+    .nunique         <frame/nunique>
+    .nunique1        <frame/nunique1>
```

Comment:
Methods should have parentheses, e.g. `.copy()`, while attributes should not, e.g. `.names`.
Seems you're missing lots of parentheses here.

Reply:
Yep, let me change it. I wonder why it doesn't fail though...